app/utils/urls.py

```diff
@@ -1,20 +1,13 @@
-from urllib.parse import urlsplit, urlunsplit
-from flask import current_app, url_for
+from urllib.parse import urlunsplit, urlencode
 
+from flask import current_app
+from notifications_utils.base64_uuid import bytes_to_base64, uuid_to_base64
 
-def get_document_download_url(service_id, document_id, key):
```

Comment:
I think maybe we should keep this function as well and add the "api_url" to the response

Reply:
as in, return to notify
```json
{
  "document_id": "...",
  "api_url": "direct link to pdf",
  "frontend_url": "link to front-end user flow",
}```

?

i guess